McCorrections/include/BTagWeight.h

```diff
@@ -67,9 +67,12 @@ class BTagWeight : public IWeightProvider {
     virtual double Get(EventInfo& event) const override;
     virtual double Get(const ntuple::ExpressEvent& /*event*/) const override;
 
-    double Get(EventInfo& event, DiscriminatorWP wp, bool use_iterative_fit, UncertaintySource unc_source,
+    double Get(EventInfo& eventInfo, DiscriminatorWP wp, bool use_iterative_fit, UncertaintySource unc_source,
                UncertaintyScale unc_scale) const;
-
+    std::map<UncertaintyScale, std::vector<float>> GetEvtWeightShifted(EventInfo& eventInfo, DiscriminatorWP wp,
```

Comment:
There is no need to add new method GetEvtWeightShifted - the code should work with minimal modifications to Get method.